package.json

```diff
@@ -12,7 +12,24 @@
     "prepack": "npm run build",
     "preversion": "kacl prerelease",
     "version": "kacl release && git add CHANGELOG.md",
-    "postversion": "git push --follow-tags && gh-release"
+    "postversion": "git push --follow-tags && gh-release",
+    "lint": "eslint '**/*.{js,jsx,ts,tsx}'",
+    "lint:fix": "eslint '**/*.{js,jsx,ts,tsx}' --fix",
+    "prettier": "prettier --check '**/*.{js,jsx,ts,tsx,json,md,html,yml,yaml}'",
+    "prettier:fix": "prettier --write '**/*.{js,jsx,ts,tsx,json,md,html,yml,yaml}'"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```

Comment:
I know that lots of people seem to reach for husky but haven't had much success with the experience myself. I have found that it adds friction while running synchronous work during commit. My experience has been that this breaks my flow more often than not and catches nothing more than what CI would.

Reply:
Was it maybe because your husky was setup wrong? I dont understant what exactly it was breaking for you. The point is that if you forget to fix a lint or format error somewhere, husky fixes it for you during commiting. It results in less failed builds and repeated commits which is a better DX.

Reply:
Hi @Kamahl19, sounds like husky has worked well for you. I'm willing to consider it as this project matures but would prefer to leave these sorts of checks in CI for now.